cookbooks/compute_conf/attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['compute_conf']['host_ip'] = ""
 default['compute_conf']['service_host'] = ""
+default['devstack_conf']['git_base'] = "https://github.com"
```

Comment:
@toshikazu0314 
Thank you for the review. 
It is somewhat matter of personal taste:)
'devstack_conf' is used in both compute_conf and controller_conf recipes.
This attribute set the GIT_BASE value to "https://github.com" in local.conf for devstack.
Original value "git://git.openstack.org" not work well behind proxy and https works fine in both proxy or no_proxy envs.
I purposely set the attribute scope to 'devstack_conf', because when you set this value in multiple multiple nodes, you could distinguish this attribute from other node related attributes.  
On other words, if you want to change the git repo for devstack, you only have to change,
"devstack_conf" => {
          "git_base" => "some other repo for devstack"
        }
for each node.
Or maybe you cold set it globally using environment file for vagrant.
However, if you change this to smt like,
default['compute_conf']['git_base'] = "https://github.com"
global definition logic will be more complex. 

Same comment goes to your next comment about `git_base` in controller_conf.